lib/navigation/consts.hpp

```diff
@@ -0,0 +1,27 @@
+#include <array>
+#include <cstdint>
+#include <optional>
+
+#include "core/types.hpp"
+
+namespace hyped::navigation {
+
+static constexpr core::Float kTrackLength       = 100.0;  // m
+static constexpr core::Float kBrakingDistance   = 20.0;   // m TODOLater:check!
+static constexpr core::Float kPi                = 3.14159265359;
+static constexpr core::Float kWheelCicumference = kPi * 0.1;  // m TODOLater: check!
+static constexpr core::Float kStripeDistance    = 10.0;       // m TODOLater:check!
+
+// define sensor checks return type
+enum class SensorChecks { kUnacceptable = 0, kAcceptable };
```

Comment:
I'm unfamiliar with this notation: what value does kAcceptable take on?

Reply:
1

Reply:
(basically like bool but more descriptive)